src/providers/generalWalletProvider.tsx

```diff
@@ -259,11 +261,15 @@ export const GeneralWalletProvider: React.FC<{
 				});
 				return;
 			case ChainType.SOLANA: {
+				const lamports = new BigNumber(value)
+					.times(LAMPORTS_PER_SOL)
+					.toFixed();
+
```

Comment:
Let's unify the convention of calling this method for both Solana and EVM and expect the caller does this calculation and pass lamports